Tasks/Decorator.py

```diff
@@ -0,0 +1,44 @@
+
+def cache(fn):
+    def wrapper(*args, **kwargs):
+        str_args = str(args)
+        str_kwards = str(kwargs) + str_args
+        if str_kwards in wrapper.cash:
+            result = wrapper.cash[str_kwards]
+        else:
+            result = fn(*args, **kwargs)
+        wrapper.cash[str_kwards] = result
```

Comment:
Чуть более эффективный способ:
```suggestion
        if str_kwards not in wrapper.cash:
            wrapper.cash[str_kwards] = fn(*args, **kwargs)
        return wrapper.cash[str_kwards]
```